dallinger/experiment_server/experiment_server.py

```diff
@@ -296,29 +294,36 @@ def advertisement():
     except exc.SQLAlchemyError:
         status = None
 
-    if status == STARTED and not debug_mode:
-        # Once participants have finished the instructions, we do not allow
-        # them to start the task again.
-        raise ExperimentError('already_started_exp_mturk')
-    elif status == COMPLETED:
+    if status == 'working' and part.end_time is not None:
```

Comment:
I think this should be `if status == 'submitted':` and we should make sure that the `/worker_complete` route updates the status to 'submitted'

Good catch on the difference in how the Dallinger participant model handles the status field.